src/view/add-trip-form.js

```diff
@@ -1,109 +1,181 @@
-export const addTripTemplate = () => {
+import {TRANSFER_TYPES} from "../const.js";
+import {ACTIVITY_TYPES} from "../const.js";
+import {PREPOSITION} from "../const.js";
+import {ACTIVITY} from "../const.js";
+
+const OFFERS_NAME = {
+  addLuggage: `Add luggage`,
+  travelByTrain: `Travel by train`,
+  switchToComfort: `Switch to comfort class`,
+  chooseSeats: `Choose seats`,
+  addMeal: `Add meal`
+};
+const CLASS_NAME = {
+  luggage: `luggage`,
+  train: `train`,
+  comfort: `comfort`,
+  seats: `seats`,
+  meal: `meal`
+};
+
+const getCreateChooseTransferTypeTemplate = (it) => {
+  return TRANSFER_TYPES.map((type) => `<div class="event__type-item">
+  <input id="event-type-${type}-1" class="event__type-input  visually-hidden" type="radio" name="event-type" value="${type}" ${it === type ? `checked` : ``}>
+  <label class="event__type-label  event__type-label--${type.toLowerCase()}" for="event-type-${type}-1">${type}</label>
+</div>`).join(``);
```

Comment:
вот так верска в строках будет читаться легче :)
```suggestion
  return TRANSFER_TYPES.map((type) => `
  <div class="event__type-item">
    <input id="event-type-${type}-1" class="event__type-input  visually-hidden" type="radio" name="event-type" value="${type}" ${it === type ? `checked` : ``}>
    <label class="event__type-label  event__type-label--${type.toLowerCase()}" for="event-type-${type}-1">${type}</label>
  </div>
`).join(``);
```